lib/exporter.go

```diff
@@ -55,13 +55,10 @@ type Exporter struct {
 
 	couchLog *prometheus.GaugeVec
 
-	fabricWorker            *prometheus.GaugeVec
-	fabricOpenShard         *prometheus.GaugeVec
-	fabricReadRepairs       *prometheus.GaugeVec
-	fabricFailure           *prometheus.GaugeVec
```

Comment:
Failure, MismatchedErrors, WriteQuorumErrors have been removed. That looks unrelated to the pull request. Could it be that I didn't notice the fields not being used when reviewing the previous pull request? 😅

Reply:
Please accept my apologies. I created these previously but ended up not using them and I neglected to clean up properly. I should have picked it up before I made the previous pull request.

Reply:
No problem, me as a reviewer hasn't seen it either in the previous PR :)
And I'm fine with removing them here.